R/appDependencies.R

```diff
@@ -43,12 +67,15 @@
 #' @inheritParams listDeploymentFiles
 #' @param appDir Directory containing application. Defaults to current working
 #'   directory.
-#' @return A data frame with columns:
+#' @returns A data frame one row for each depedency (direct and indirect) and
+#'   4 columns:
+#'
 #'   * `Package`: package name.
 #'   * `Version`: local version.
-#'   * `Source`: where the package was installed from.
-#'   * `Repository`: for CRAN and CRAN-like repositories, the url to the
-#'      repo.
+#'   * `Source`: where the package was installed from, as described above.
+#'   * `Repository`: for CRAN and CRAN-like repositories, the URL to the
```

Comment:
The fact that _THIS_ `Repository` field is a URL, while the package description `Repository` is a _name_ has certainly caused confusion!